articles/storage/blobs/storage-dotnet-how-to-use-blobs.md

```diff
@@ -20,385 +20,370 @@ ms.author: tamram
+## Get the sample project
 
-### About this tutorial
-This tutorial shows how to write .NET code for some common scenarios using Azure Blob storage. Scenarios covered include uploading, listing, downloading, and deleting blobs.
+To begin, [clone the sample project from GitHub](https://github.com/cartermp/TutorialForStorage). The rest of the tutorial will work with this project.
 
-**Prerequisites:**
+If you're unfamiliar with git, you can also download the project as a zip file. In the GitHub repository, there is a green button called **Clone or download**. You can click that and select **Download ZIP**:
 
-* [Microsoft Visual Studio](https://www.visualstudio.com/)
-* [Azure Storage Client Library for .NET](https://www.nuget.org/packages/WindowsAzure.Storage/)
-* [Azure Configuration Manager for .NET](https://www.nuget.org/packages/Microsoft.WindowsAzure.ConfigurationManager/)
-* An [Azure storage account](../common/storage-create-storage-account.md?toc=%2fazure%2fstorage%2fblobs%2ftoc.json#create-a-storage-account)
+![Screenshot of GitHub showing how to clone or download the sample application](media/storage-blobs-introduction/download-sample.png)
 
-[!INCLUDE [storage-dotnet-client-library-version-include](../../../includes/storage-dotnet-client-library-version-include.md)]
+## Set up your environment
 
-### More samples
-For additional examples using Blob storage, see [Getting Started with Azure Blob Storage in .NET](https://azure.microsoft.com/documentation/samples/storage-blob-dotnet-getting-started/). You can download the sample application and run it, or browse the code on GitHub.
+Ensure that you have [Visual Studio 2017](https://www.visualstudio.com/vs/) installed with the **ASP.NET and web development** and **Azure Development** workloads. This will give you all of the tools that you will need for this tutorial.
 
-[!INCLUDE [storage-blob-concepts-include](../../../includes/storage-blob-concepts-include.md)]
+![Screenshot of searching for the Azure Storage Emulator via Windows search](media/storage-blobs-introduction/workloads.png)
 
-[!INCLUDE [storage-create-account-include](../../../includes/storage-create-account-include.md)]
+## Run the application locally
 
-[!INCLUDE [storage-development-environment-include](../../../includes/storage-development-environment-include.md)]
+First, start the [Azure Storage Emulator](https://docs.microsoft.com/azure/storage/common/storage-use-emulator) by pressing the **Windows** key on your keyboard and searching for **Azure Storage Emulator**:
 
-### Add using directives
-Add the following **using** directives to the top of the `Program.cs` file:
+![Screenshot of searching for the Azure Storage Emulator via Windows search](media/storage-blobs-introduction/storage-emulator.png)
```

Comment:
Is this available on other platforms as well?

Reply:
Nope, unfortunately.

Reply:
We may want to point out for anyone that is trying to develop on Mac or Linux. Seems like that should be a focus too so everyone can use this.